GA/HeaderFiles/ConstraintHPol.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CONSTRAINTHPOL_H
+#define CONSTRAINTHPOL_H
+
+bool ConstraintHPol(float radius, float length)
+{
+  // Set variables
+  bool intersect = true;
+
+  // distances in cm
+  float max_radius = 10.0;
+  float min_radius = 0.25;
+  float max_length = 10000.0;
+  float min_length = 200.0;
+
+  // Check to see if antenna meets requirements
+  if (radius > max_radius || radius < min_radius || length > max_length || length < min_length)
+  {
+    intersect = true;
+  }
+  else
+  {
+    intersect = false;
+  }
+
+  // Return interesect condition
+  return intersect;
+}
+
+#endif
```

Comment:
There should be an empty line at the end of the file.  You should be able to configure your editor to add it automatically when you save the file (if it's not there already).  Checking the option to remove trailing whitespace on save is also a good idea.

We also discussed coding conventions regarding indentation and `{}` placements, but we still don't have a system in place to check/fix it automatically.